content/en/docs/test.md

```diff
@@ -297,7 +296,8 @@ tables, use HTML instead.
 
 ## Visualizations with Mermaid
 
-Add `mermaid: true` to the [front matter](https://gohugo.io/content-management/front-matter/) of any page to enable [Mermaid JS](https://mermaidjs.github.io) visualizations. The Mermaid JS version is specified in [/layouts/partials/head.html](https://github.com/kubernetes/website/blob/master/layouts/partials/head.html)
+You can use [Mermaid JS](https://mermaidjs.github.io) visualizations.
```

Comment:
Some feedback:
Indicate that `mermaid.js` is loaded by including a custom Hugo shortcode named `mermaid`. For example:

You can create [Mermaid JS] (...) diagrams and flowcharts by including the
the `mermaid` shortcode in a content page. The `mermaid` shortcode accepts the mermaid.js markup (instructions)
to render a visualization on an HTML page.
nit: Is the mermaid JS version set in the comments in `layouts/partials/head.html`?

Reply:
MermaidJS is now vendored, but ` content/en/docs/test.md` doesn't make that clear (wasn't updated when it changed). I see fixing that as something for a separate PR.

Reply:
I would prefer a bit more introduction to the mermaid shortcode (instead of "You can use").
However, this does not detract from the changes.
/lgtm